packages/cdktf-cli/test/lib/dependencies/prebuilt-providers.test.ts

```diff
@@ -184,7 +184,7 @@ describe("prebuilt-providers", () => {
         .replyWithError({ code: "ETIMEDOUT" });
 
       await expect(
-        getPrebuiltProviderVersion(
+        getPrebuiltProviderVersions(
```

Comment:
Do these tests need an update? The function returns an array now, doesn't it?

Reply:
They might also need new test cases / adjusted mocks for the newly added requests?